public/components/common/form/index.tsx

```diff
@@ -7,13 +7,12 @@ import { InputFormSwitch } from './input_switch';
 import { InputFormFilePicker } from './input_filepicker';
 import { InputFormTextArea } from './input_text_area';
 import { EuiFlexGroup, EuiFlexItem, EuiFormRow } from '@elastic/eui';
-import { OsCard } from '../../../controllers/register-agent/components/os-card/os-card';
-import { SettingTypes } from './types';
 
-interface InputFormProps {
-  type: SettingTypes;
+export interface InputFormProps {
+  type: string;
```

Comment:
**thought**: if `SettingTypes` is the set of allowed values, then we should it instead of the generic `string` type.